slack_wrapper.sh

```diff
@@ -9,11 +9,9 @@ echo "SLACK_CHANNEL=$SLACK_CHANNEL"
 
 send_slack ()
 {
-  curl -X POST --data-urlencode 'payload={"channel": "'"$SLACK_CHANNEL"'", "username": "Nelson", "text": "'"$1"'\n`'"$CMD"'`", "icon_emoji": ":nelson:"}' $SLACK_URL
+  curl -X POST --data-urlencode 'payload={"channel": "'"$SLACK_CHANNEL"'", "username": "Nelson", "text": "'"$1"'\n`'"$CMD"'`", "icon_emoji": ":nelson:"}' $SLACK_URL --insecure
```

Comment:
The root CA on the host is not up to date (jessie host) => not maintained anymore